packages/rudy/src/core/createRouter.js

```diff
@@ -132,7 +131,7 @@ export default (
   }
 
   return {
-    ...api,
```

Comment:
This was a problem because the extra properties (`getLocation` etc) that are assigned to the `api` object did not also get added to the returned object here.

Reply:
Ahh actually, we just went over this yesterday in Remixx. As long as we are still able to access the internal API, which is seems we are

Reply:
yes, well in fact before it was not completely possible because this line:
https://github.com/respond-framework/rudy/blob/32979d1bcef5f6847b628b7bf0954bb861e36f04/packages/rudy/src/core/createRouter.js#L99-L100
executed after the return statement. With this change the API is still incomplete when returned, but it's filled in later so it works for most purposes.